### ISSUE TITLE
pixel grid does not appear until mouseover event

### ISSUE THREAD

**Issue by michaelpaulhirsch** [javierluraschi/pixels#5]
Hi Javier,
Thanks for an awesome package!
I noticed that the pixel grid doesn't appear until the user moves the mouse over the box that the pixel is in. I've noticed this consistently inside shiny apps and in the viewer on different machines (ubuntu & redhat).

Is there a patch or workaround for this?

Thanks

**Comment by mattwarkentin** [comment]
Yes, I have noticed this as well. Would be great to have the grid appear right when the shiny app is loaded rather than requiring a mouse-over.
